Python/Tests/TestAdapterTests/TestExecutorTests.cs

```diff
@@ -211,7 +213,7 @@ public void RunPytestParameterizedAndDiscovery() {
             );
             var discoveryContext = new MockDiscoveryContext(runSettings);
             var discoverySink = new MockTestCaseDiscoverySink();
-            var logger = new MockMessageLogger();
+            var logger = new MockMessageLogger(TestContext); // pass TestContext
```

Comment:
The comments here and elsewhere about // pass TestContext seem really redundant.